django/contrib/redirects/migrations/0003_migrate_sites.py

```diff
@@ -0,0 +1,31 @@
+from django.db import migrations
+
+
+def migrate_site_domains(apps, schema_editor):
+    try:
+        Site = apps.get_model('sites', 'Site')
+    except LookupError:
+        return
+
+    Redirect = apps.get_model('redirects', 'Redirect')
+
+    db_alias = schema_editor.connection.alias
+    for r in Redirect.objects.using(db_alias).all():
```

Comment:
This migration will be very slow. I would recommend using the `F` object to perform the migration in a single update statement rather than looping over all rows.

Reply:
As far as I understand (and tried) F object cannot be used to update from related field. If you'll be able to do it, feel free to send working example to me.

Reply:
Dang, you are right. I forgot.
Can you use the `iterator` than, to avoid migrations going out of memory?